lib/diagnostic.js

```diff
@@ -6,20 +6,22 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+
+let words = paragraph.split(' ');
+const normalizeWords = function (paragraph) {
+  return word.replace(/\W+/,'').toUpperCase();
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (paragraph) {
+  return normalizedWords.map(paragraph);
 };
 
```

Comment:
You need to be careful about what you name things and what the object you're method chaining is! Is normalizedWords and array or a function? What does it return?